boards/adi/max32657evkit/doc/index.rst

```diff
@@ -0,0 +1,605 @@
+|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
+|           |               | | Open      | |  | Disable EXT SWD Clock Connection.                                             |               |
+|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
+|           |               |               |                                                                                                  |
++-----------+---------------+---------------+--------------------------------------------------------------------------------------------------+
+
+
+Zephyr board options
+********************
+
+The MAX32657 microcontroller (MCU) is an advanced system-on-chip (SoC)
+featuring an ARM Cortex-M33 architecture that provides Trustzone technology
+which allow define secure and non-secure application.
+Zephyr provides support for building for both Secure and Non-Secure firmware.
+
+The BOARD options are summarized below:
+
++-------------------------------+-------------------------------------------+
+| BOARD                         | Description                               |
++===============================+===========================================+
+| max32657evkit/max32657        | For building Trust Zone Disabled firmware |
++-------------------------------+-------------------------------------------+
+| max32657evkit/max32657/ns     | For building Non-Secure firmware          |
++-------------------------------+-------------------------------------------+
+
+
+BOARD: max32657evkit/max32657
+=============================
+
+Build the zephyr app for max32657evkit/max32657 board will generate secure firmware
+for zephyr. In this configuration 960KB of flash is used to store the code and 64KB
+is used for storage section. In this mode tf-m is off and secure mode flag is on
+``CONFIG_TRUSTED_EXECUTION_SECURE=y`` and ``CONFIG_BUILD_WITH_TFM=n``
+
++----------+------------------+---------------------------------+
+| Name     | Address[Size]    | Comment                         |
++==========+==================+=================================+
+| slot0    | 0x1000000[960k]  | Secure zephyr image             |
++----------+------------------+---------------------------------+
+| storage  | 0x10f0000[64k]   | File system, persistent storage |
++----------+------------------+---------------------------------+
+
+Here are the instructions to build zephyr with a non-secure configuration,
+using :zephyr:code-sample:`blinky` sample:
+
+   .. code-block:: bash
+
+      $ west build -b max32657evkit/max32657 samples/basic/blinky/ -p
+
+
+BOARD: max32657evkit/max32657/ns
+================================
+
+The max32657evkit/max32657/ns board configuration is used to build the secure firmware
```

Comment:
board configuration does not exist, it is board target, and put board target in \`\`

Reply:
Done.